src/fread.c

```diff
@@ -2250,43 +2232,43 @@ int freadMain(freadMainArgs _args) {
   //*********************************************************************************************
   // [11] Read the data
   //*********************************************************************************************
-  bool stopTeam=false, firstTime=true, restartTeam=false;  // bool for MT-safey (cannot ever read half written bool value badly)
-  int nTypeBump=0, nTypeBumpCols=0;
-  double tRead=0, tReread=0;
-  double thRead=0, thPush=0;  // reductions of timings within the parallel region
-  int max_col=0;
-  char *typeBumpMsg=NULL;  size_t typeBumpMsgSize=0;
-  int typeCounts[NUMTYPE];  // used for verbose output; needs populating after first read and before reread (if any) -- see later comment
+  bool stopTeam = false, firstTime = true, restartTeam = false;  // bool for MT-safey (cannot ever read half written bool value badly)
+  int nTypeBump = 0, nTypeBumpCols = 0;
+  double tRead = 0, tReread = 0;
+  double thRead = 0, thPush = 0;  // reductions of timings within the parallel region
+  int max_col = 0;
+  char *typeBumpMsg = NULL; size_t typeBumpMsgSize = 0;
+  int typeCounts[NUMTYPE]; // used for verbose output; needs populating after first read and before reread (if any) -- see later comment
   #define internalErrSize 1000
   char internalErr[internalErrSize+1]="";  // must be compile time size: the message is generated and we can't free before STOP
   int64_t DTi = 0;                  // the current row number in DT that we are writing to
   const char *headPos = pos;       // the jump start corresponding to DTi
   int nSwept = 0;                  // count the number of dirty jumps that were swept
   const char *quoteRuleBumpedCh = NULL;   // in the very rare event of an out-of-sample quote rule bump, give a good warning message
   int64_t quoteRuleBumpedLine = -1;
-  int buffGrown=0;
+  int buffGrown = 0;
   // chunkBytes is the distance between each jump point; it decides the number of jumps
   // We may want each chunk to write to its own page of the final column, hence 1000*maxLen
   // For the 44GB file with 12875 columns, the max line len is 108,497. We may want each chunk to write to its
   // own page (4k) of the final column, hence 1000 rows of the smallest type (4 byte int) is just
   // under 4096 to leave space for R's header + malloc's header.
-  size_t chunkBytes = umax((uint64_t)(1000*meanLineLen), 1ULL/*MB*/ *1024*1024);
+  size_t chunkBytes = umax((uint64_t)(1000 * meanLineLen), 1ULL * 1024 * 1024/*MB*/);
```

Comment:
pedantic suggestion 😉 

```suggestion
  size_t chunkBytes = umax((uint64_t)(1000 * meanLineLen), 1ULL * 1024 * 1024/*MiB*/);
```